src/agent/Cargo.toml

```diff
@@ -61,6 +61,13 @@ vsock-exporter = { path = "vsock-exporter" }
 serde = { version = "1.0.129", features = ["derive"] }
 toml = "0.5.8"
 
+# Image pull/decrypt
+oci-distribution = "0.7"
```

Comment:
- I understand why this is being specified as a partial semver, but all other versions are specified in full semver format, so for consistency this should be too, or a comment added to explain why [*].
- Can you update to the latest version is `"0.8.0"`?

[*] - I wonder if we need to discuss this topic at the AC tbh. However, once we get all rust code to pass `cargo audit` we will introduce a GitHub action to ensure we stay current so full specification shouldn't be an issue.

Reply:
Yes, binary shoud use full semver, I changed the version to `0.7.0`, since ocicrypt-rs also have depdency for `oci-distribution` and we not test with `0.8.0` yet, we will update them at the same time